lib/kennel/predicate_accessors.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Kennel
+  module PredicateAccessors
+    def self.included(into)
+      into.extend(::Kennel::PredicateAccessors::ClassMethods)
+    end
+
+    module ClassMethods
+      def attr_reader(*symbols)
```

Comment:
maybe simpler and not risk of anything going wrong :)
```suggestion
      def attr_reader?(*symbols)
```

Reply:
... or doing this inline
```ruby
[x].each { |s| define_method s }
```

Reply:
Renamed, c36c8e1d6e3582ba0cf02ec8673c55dff92b245a